google/cloud/security/common/gcp_api/compute.py

```diff
@@ -47,6 +47,33 @@ def __init__(self, credentials=None, version=None):
     # TODO: Migrate helper functions from gce_firewall_enforcer.py
     # ComputeFirewallAPI class.
 
+    def get_backend_services(self, project_id):
+        """Get the backend services for a project.
+
+        Args:
+            project_id: The project id.
+
+        Yield:
+            An iterator of backend services for this project.
+
+        Raise:
+            api_errors.ApiExecutionError if API raises an error.
+        """
+        backend_services_api = self.service.backendServices()
+        list_request = backend_services_api.aggregatedList(
+            project=project_id)
+        list_next_request = backend_services_api.aggregatedList_next
+
+        try:
```

Comment:
There's a util here to [build a list of paged results](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/google/cloud/security/common/gcp_api/_base_client.py#L104) that may be of use.